src/pages/Login/SignatureModal/index.tsx

```diff
@@ -42,19 +42,13 @@ export const SignatureModal: React.FC<{
       .catch(() => {
         setSigning(false)
       })
-      .finally(() => setSigning(false))
+      .finally(() =>
+        setTimeout(() => {
+          setSigning(false)
+        }, 5000)
```

Comment:
Closing the modal & `setSigning(false)` was not in sync, I added this so that the user won't see any changes in the UI. 

```
.finally(() => {
  signatureModalService.closeModal()
  setTimeout(() => {
    setSigning(false)
  }, 300)
})

Now they are in sync. Added a small time-out so that there are no UI changes when the modal is animating out.